src/content/reading_material/_index.en.md

```diff
@@ -13,9 +13,10 @@ There are multiple presentations/demo recordings on Submariner available online.
 2. KubeCon North America 2019 [slides](https://static.sched.com/hosted_files/kccncna19/7d/Submariner%20Kubecon%20NA%202019%20%281%29.pdf) and [video recording](https://www.youtube.com/watch?v=jMvuchSMCKU&list=PLj6h78yzYM2NDs-iu8WU5fMxINxHXlien)
 
 ## Demo Recordings
-1. [Deploying Submariner with subctl](https://www.youtube.com/watch?v=cInmBXuZsU8&t=17s)
-2. [Connecting hybrid Kubernetes clusters using Submariner](https://www.youtube.com/watch?v=fMhZRNn0fxQ)
-3. [Cross-cluster service discovery in Submariner using Lighthouse](https://www.youtube.com/watch?v=tXsemQPNhyQ)
+1. [Submariner in 60s](https://www.youtube.com/watch?v=cInmBXuZsU8)
```

Comment:
Is this the intended link - it points to a 7 min "Deploying Submariner with subctl" video?

Reply:
Yikes, good catch, thanks! Fixed in the latest.